testsuite/MDAnalysisTests/lib/test_util.py

```diff
@@ -931,6 +931,10 @@ def test_get_extention(self, extention):
 
         assert a == 'file'
         assert b == extention.lower()
+    
+    def test_get_extension_without_extension(self):
```

Comment:
good that you added a test but `get_ext` should not raise a ValueError so the test is not necessary